internal/service/docdb/sweep.go

```diff
@@ -23,6 +23,212 @@ func init() {
 			// "aws_docdb_cluster",
 		},
 	})
+	resource.AddTestSweepers("aws_docdb_subnet_group", &resource.Sweeper{
+		Name:         "aws_docdb_subnet_group",
+		F:            sweepDBSubnetGroups,
+		Dependencies: []string{},
```

Comment:
If there are any instances in the subnet group, you'll get the error

> InvalidDBSubnetGroupStateFault: Cannot delete the subnet group 'test' because at least one database instance: docdb-2022-08-18-20-48-242 is still using it.

```suggestion
		Dependencies: []string{
			"aws_docdb_cluster_instance",
		},
```